components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -496,6 +500,30 @@ - (void)setSelectedItem:(UITabBarItem *)selectedItem animated:(BOOL)animated {
   }
 }
 
+- (void)setItemsTitleVisibleTopPadding:(CGFloat)itemsTitleVisibleTopPadding {
+  if (_itemsTitleVisibleTopPadding == itemsTitleVisibleTopPadding) {
+    return;
+  }
+  _itemsTitleVisibleTopPadding = itemsTitleVisibleTopPadding;
+  for (NSUInteger i = 0; i < self.items.count; i++) {
```

Comment:
Please iterate over `itemViews` directly.  Same in `-setItemsContentVerticalMargin:`